tools/deploy_assisted_installer_configmap.py

```diff
@@ -73,15 +73,17 @@ def main():
                         "AGENT_DOCKER_IMAGE": "assisted-installer-agent"}
             for env_var_name, image_short_name in versions.items():
                 versions[env_var_name] = deployment_options.get_image_override(deploy_options, image_short_name, env_var_name)
-                log_image_revision(versions[env_var_name])
+                if deploy_options.apply_manifest:
+                    log_image_revision(versions[env_var_name])
```

Comment:
We want to log these git revisions always. Not only if a manifest was applied.

Reply:
I had added these because the image revision logging was not working inside the container build. 
```
/usr/bin/docker-current: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?.
See '/usr/bin/docker-current run --help'.
[WARNING] 2021-03-09 13:24:54,076 - deploy-service-configmap - failed to pull image quay.io/ocpmetal/assisted-service:latest inspect data, error: b''
```
In the build case, I guess it would be ok to let the logging fail with warning.

Reply:
It's failing on another issue in Prow. @oshercc is looking into that.